### ISSUE TITLE
Instruction for installation 

### ISSUE THREAD

**Issue by BowonKim** [ytdl-org/youtube-dl#28987]
## Checklist

- [ x] I'm reporting a feature request
- [x ] I've verified that I'm running youtube-dl version **2021.04.26**
- [x ] I've searched the bugtracker for similar feature requests including closed ones

## Description

I followed the instruction for installation on Windows which is downloading .exe file and I found that it doesn't work successfully if I use youtube-dl by downloading .exe file. I could solve this problem by updating  the youtube-dl using with pip command 
pip install --upgrade youtube-dl

so I would like to suggest updating the instruction for installing to install youtube-dl using with pip command.

**Comment by rautamiekka** [comment]
Did you read the readme ?

**Comment by BowonKim** [comment]
yes I did. 

Windows users can download an .exe file and place it in any location on their PATH except for %SYSTEMROOT%\System32 (e.g. do not put in C:\Windows\System32).

You can also use pip:

sudo -H pip install --upgrade youtube-dl
This command will update youtube-dl if you have already installed it. See the pypi page for more information.

This is the instruction for installation on readme and I followed the first paragraph. But I couldn't use it before upgrading the youtube-dl using with pip command. My other friends also had the same situation so I just would like to suggest to recommend to Windows users to download with pip.

**Comment by rautamiekka** [comment]
That's weird, gotta say. I just tried 2021.04.26 (the now latest) EXE and it booted just fine; I use the PIP one since I code in Python anyway.

How does it manifest you can't use the EXE ?

**Comment by BowonKim** [comment]
I and my two friends downloaded EXE but it didn't work. we also couldn't run the unit test as well( we typed this command: python -m unit test discover python test/test_download.py).  So we upgraded it using the pip command, and it works well. So we assume EXE was the issue. I think we used the previous version but not sure which version we downloaded. 

As I said, I just want to suggest updating the instruction to give another option for new people or someone who has a problem with EXE like me and my friends to download youtube-dl using the pip command.
